WalletWasabi/Helpers/PasswordHelper.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace WalletWasabi.Helpers
+{
+	public static class PasswordHelper
+	{
+		public static string[] GetPossiblePasswords(string originalPassword)
+		{
+			List<string> possiblePasswords = new List<string>()
+			{
+				originalPassword,
+				StringCutIssue(originalPassword)
```

Comment:
Q: should we do this for all the OS or only for OSX?

Reply:
Yes I agree. I fixed it.